helm-chart/dash0-operator/templates/operator/pre-delete-hook.yaml

```diff
@@ -17,7 +17,8 @@ spec:
     metadata:
       name: {{ .Release.Name }}-pre-delete-job
       labels:
-        app.kubernetes.io/instance: pre-delete-hook
+        app.kubernetes.io/name: dash0-operator
+        app.kubernetes.io/component: pre-delete
```

Comment:
q: should this component then also have the name `uninstallation-process`? (Since you chose to use that name for the associated job in [line 8](https://github.com/dash0hq/dash0-operator/pull/88/files#diff-8a97386e49994bbc6b1689d65105a3e9d3390e28118789de15c4acd88053ed63R8).

Reply:
Good catch, 904ec59